spark/src/main/scala/com/alibaba/graphar/VertexInfo.scala

```diff
@@ -0,0 +1,139 @@
+package org.alibaba.graphar
+
+import java.io.{File, FileInputStream}
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.constructor.Constructor
+import scala.beans.BeanProperty
+
+class VertexInfo() {
+  @BeanProperty var label: String = ""
+  @BeanProperty var chunk_size: Long = 0
+  @BeanProperty var prefix: String = ""
+  @BeanProperty var property_groups = new java.util.ArrayList[PropertyGroup]()
+  @BeanProperty var version: String = ""
+
+  val REGULAR_SEPERATOR = "_"
```

Comment:
move "REGULAR_SEPERATOR" into "GeneralParams" maybe better

Reply:
yeeap, nice suggestion. The "REGULAR_SEPERATOR" has been moved to `GeneralParams`